flutter_barcode_reader_web/pubspec.yaml

```diff
@@ -0,0 +1,33 @@
+name: barcode_scan_web
+description: A flutter plugin for scanning 2D barcodes and QRCodes via camera.
+version: 2.0.1
+homepage: https://github.com/mintware-de/flutter_barcode_reader
+
+dependencies:
+  flutter:
+    sdk: flutter
+  flutter_web_plugins:
+    sdk: flutter
+  js: ^0.6.0
+  barcode_scan:
+    path: ../ 
```

Comment:
Is this leftover from development? It's causing problems during `pub get` for me:
```sh
Running "flutter pub get" in flutter...                         
Error on line 13, column 11: Invalid description in the "barcode_scan_web" pubspec on the "barcode_scan" dependency: "../" is a relative path, but this isn't a local pubspec.

   ╷

13 │     path: ../ 

   │           ^^^

   ╵
pub get failed (65;    ╵)
```

Reply:
Yes, but I still haven't figured out a way to have this work without it. The reason this is needed is the fact that I needed to modify some code in the general plugin code (platform-wrapper specifically) and so I need to use that modified version of the plugin. I'm not that experienced into Dart, so if you could shed some light on this I would really appreciate it :)